### ISSUE TITLE
Add OccurrenceMappings to institutions and collections page

### ISSUE THREAD

**Issue by marcos-lg** [gbif/registry-console#546]
So far we can only add these mappings through the api but it'd be useful to show them in the UI so we can notice them easily.

These are the fields that an occurrence mapping can have(only the dataset key is mandatory): https://github.com/gbif/gbif-api/blob/master/src/main/java/org/gbif/api/model/collections/OccurrenceMapping.java

Explanation about it can be found here: https://github.com/gbif/registry/blob/dev/registry-service/README.md

Example in DEV: https://api.gbif-dev.org/v1/grscicoll/collection/cf09a9c6-c9e9-4c32-b23b-b913ac702b04/occurrenceMapping
Example in UAT: https://api.gbif-uat.org/v1/grscicoll/collection/d5542a80-fa1b-45d0-995f-141a42b6be1e/occurrenceMapping

**Comment by MortenHofft** [comment]
I didn't know we had that option. Nice. What happens if there are conflicting rules? Are there any flags connected to this that we should show?

**Comment by marcos-lg** [comment]
If there are rules that make the lookup find more than 1 result it will return an `AMBIGUOUS_EXPLICIT_MAPPINGS` status. It's the same as for the exact and fuzzy matches. So this is the only new flag that is added here. 

Maybe we can add a link to the documentation(not sure which one).

**Comment by MortenHofft** [comment]
Is that an occurrence flag that isn't added yet? https://api.gbif-uat.org/v1/enumeration/basic/OccurrenceIssue

**Comment by marcos-lg** [comment]
No, I meant that flag is from the lookup ws. The occurrence flag would be `AMBIGUOUS_INSTITUTION` or `AMBIGUOUS_COLLECTION`. There are no changes in the flags.
